This is needed because busybox ash (our /bin/sh) doesn't spawn a
subprocess for the `sleep` command.

--- a/pueue_lib/src/process_helper/unix.rs
+++ b/pueue_lib/src/process_helper/unix.rs
@@ -97,7 +97,7 @@
         // Get all child processes, so we can make sure they no longer exist afterwards.
         // The process group id is the same as the parent process id.
         let group_pids = get_process_group_pids(pid);
-        assert_eq!(group_pids.len(), 3);
+        assert_eq!(group_pids.len(), 2);
 
         // Kill the process and make sure it'll be killed.
         assert!(kill_child(0, &mut child).is_ok());
@@ -130,7 +130,7 @@
         // Get all child processes, so we can make sure they no longer exist afterwards.
         // The process group id is the same as the parent process id.
         let group_pids = get_process_group_pids(pid);
-        assert_eq!(group_pids.len(), 3);
+        assert_eq!(group_pids.len(), 2);
 
         // Kill the process and make sure it'll be killed.
         send_signal_to_child(&mut child, Signal::SIGKILL).unwrap();
